dapps/marketplace/src/components/WaitForTransaction.js

```diff
@@ -128,6 +129,7 @@ class WaitForTransaction extends Component {
           let content
           if (error) {
             console.error(error)
+            Sentry.captureException(error)
```

Comment:
Does the <Query /> component from `react-apollo` return errors as something that `Sentry.captureException` won't choke on?

Reply:
It looks [like it extends](https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/src/errors/ApolloError.ts) `Error` so it should be OK. Not sure if you can verify this easily.

Reply:
Yeah, was worried about that too but it returns an `ApolloEror` which extends `Error`.  Should be fine